src/main/java/com/yahoo/bullet/querying/RunningQuery.java

```diff
@@ -5,64 +5,22 @@
  */
 package com.yahoo.bullet.querying;
 
-import com.yahoo.bullet.common.BulletConfig;
-import com.yahoo.bullet.common.BulletError;
-import com.yahoo.bullet.common.Initializable;
-import com.yahoo.bullet.parsing.Parser;
-import com.yahoo.bullet.parsing.Query;
+import com.yahoo.bullet.query.Query;
 import lombok.Getter;
-
-import java.util.List;
-import java.util.Optional;
+import lombok.RequiredArgsConstructor;
 
 /**
  * A wrapper for a running query.
  */
-public class RunningQuery implements Initializable {
```

Comment:
Will removing this break something in Spark? It might need to start the query separately? Probably should verify